packages/fast-form-generator-react/src/form/form-item.children.tsx

```diff
@@ -444,6 +498,16 @@ class FormItemChildren extends React.Component<IFormItemChildrenProps & IManaged
         });
     }
 
+    /**
+     * Handle clicking on the window - closes the Select if the users
+     * is clicking somewhere other than the menu.
+     */
+    private handleWindowClick = (e: any): void => {
```

Comment:
same comment as before, can be more precise typing than "any"

Reply:
Yeah I originally tried this but it didn't like it. Will look into more.